packages/front-end/hooks/useExperimentStatusIndicator.ts

```diff
@@ -13,57 +13,86 @@ import {
 } from "back-end/types/experiment";
 import useOrgSettings from "@/hooks/useOrgSettings";
 import { useUser } from "@/services/UserContext";
-import useApi from "@/hooks/useApi";
+import { useDefinitions } from "@/services/DefinitionsContext";
 
-export function useExperimentDecisionCriteria() {
-  const settings = useOrgSettings();
-  const decisionCriteria = !settings?.defaultDecisionCriteriaId
-    ? PRESET_DECISION_CRITERIA
-    : PRESET_DECISION_CRITERIAS.find(
-        (dc) => dc.id === settings.defaultDecisionCriteriaId
-      );
-  const { data } = useApi<{ decisionCriteria: DecisionCriteriaInterface }>(
-    `/decision-criteria/${settings?.defaultDecisionCriteriaId}`,
-    {
-      shouldRun: () =>
-        !!settings?.defaultDecisionCriteriaId && !decisionCriteria,
-    }
+export function getExperimentDecisionCriteria({
```

Comment:
nit: Do we need the `export` here? Or should this be more of an internal function?